src/site/content/en/blog/iframe-lazy-loading/index.md

```diff
@@ -0,0 +1,305 @@
+interested in the [YouTube lite
+component](https://github.com/paulirish/lite-youtube-embed).
+{% endAside %}
+
+**Lazy-loading Instagram embeds (saves >100KB gzipped on initial load):**
+
+Instagram embeds provide a block of markup and a script, which injects an
+iframe into your page. Lazy-loading this iframe avoids having to load all of
+the script necessary for the embed. Given such embeds are often displayed below
+the viewport in most articles, this seems like a reasonable candidate for
+native lazy-loading of their iframe.
+
+**Lazy-loading Spotify embeds (saves 514KB on initial load):**
+
+```html
+<iframe src="https://open.spotify.com/embed/album/1DFixLWuPkv3KT3TnV35m3" 
+        loading="lazy"
+        width="300" 
+        height="380" 
+        frameborder="0" 
+        allowtransparency="true" 
+        allow="encrypted-media"></iframe>
+```
+
+Although the above embeds illustrate the potential benefits to lazy-loading
+iframes for media content, there's the potential to also see these benefits for
+ads.  
+
+### Case study: Natively lazy-loading the Facebook's social plugins
+
+Facebook's _social plugins_ allow developers to embed Facebook content in their
+web pages. There's a number of these plugins offered, such as embedded posts,
+photos, videos, comments… The most popular is the [Like
+plugin](https://developers.facebook.com/docs/plugins/like-button/) - a button
+that shows a count of who has "liked" the page. By default, embedding the Like
+plugin in a webpage (using the FB JSSDK) pulls in ~215KB of resources, 197KB of
+which is JavaScript. In many cases, the plugin may appear at the end of an
+article or near the end of a page, so loading it eagerly when it's offscreen
+may be suboptimal.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="./fblike.png" alt="Facebook Like Button">
+</figure>
+
+Thanks to engineer Stoyan Stefanov, [all of Facebook's social plugins now
+support native iframe
+lazy-loading](https://developers.facebook.com/docs/plugins/like-button#settings).
+Developers who opt in to lazy-loading via the plugins' `data-lazy`
+configuration will now be able to avoid it loading until the user scrolls
+nearby. This enables the embed to still fully function for users that need it,
+while offering data-savings for those who are not scrolling all the way down a
+page. We are hopeful this is the first of many embeds to explore native iframe
+lazy-loading in production.
+
+### Wait, can't browsers just automatically lazy-load offscreen iframes?
+
+They certainly can. In Chrome 77, Chrome added support for automatically
+natively lazy-loading offscreen images and iframes when a user has opted into
+[Lite Mode](https://blog.chromium.org/2019/04/data-saver-is-now-lite-mode.html)
+(Data Saver mode) in Chrome for Android. 
+
+Lite Mode users are commonly in regions of the world where network connection
+quality and data savings plans are not the greatest. Every byte matters and so
```

Comment:
Should it be "Lite Mode is commonly used in ..." and remove "savings"?

Reply:
That's a good suggestion! I'll update to account for this shortly.